content/docs/connect/connectivity-issues.md

```diff
@@ -97,9 +97,9 @@ Native client libraries:
 | postgres.js       | JavaScript  | no                                                       |
 | pgmoon            | Lua         |                                                          |
 | asyncpg           | Python      | yes                                                      |
-| pg8000            | Python      | [depends](https://github.com/neondatabase/neon/pull/2008) |
+| pg8000            | Python      | yes (as of pg8000 v1.29.3)                               |
```

Comment:
I've double-checked why the previous version we used didn't work, and it turned out the problem was in the `scramp` library (a dependency of `pg8000`).  The bug with channel binding was fixed in `scramp>=1.4.3`, so now even `pg8000==1.29.1` will work (if `scramp>=1.4.3`), the later version of `pg8000` has such `scramp` version restriction